skills/cavendish/references/commands.md

```
@@ -0,0 +1,42 @@
+| 0 | Success | — |
+| 2 | `cdp_unavailable` | `cavendish init` |
+| 3 | `chrome_not_found` | Install Google Chrome |
+| 4 | `auth_expired` | Re-login in Chrome |
+| 5 | `cloudflare_blocked` | Solve in Chrome tab |
+| 6 | `selector_miss` | `cavendish status` |
+| 7 | `timeout` | Increase `--timeout` |
+| 8 | `chrome_launch_failed` | `cavendish init` |
+| 9 | `chrome_close_failed` | Close Chrome manually |
+
+JSON error output (stderr, with `--format json`):
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)